tests/test_output_watchdog_runtime.sh

```
@@ -0,0 +1,105 @@
+
+if [ "$ELAPSED_SECONDS" -gt 10 ]; then
+  echo "FAIL: watchdog took ${ELAPSED_SECONDS}s to terminate fatal crash output" >&2
+  exit 1
+fi
+
+if ! grep -Fq "Fatal output detected" "$OUTPUT_FILE"; then
+  echo "FAIL: watchdog output did not explain fatal crash termination" >&2
+  exit 1
+fi
+
+>"$OUTPUT_FILE"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)